src/main/io/ledstrip.c

```diff
@@ -926,15 +926,15 @@ static void applyRainbowLayer(bool updateNow, timeUs_t *timer)
 
     if (updateNow) {
         offset += ledStripConfig()->ledstrip_rainbow_freq;
-        *timer += HZ_TO_US(TASK_LEDSTRIP_RATE_HZ);
+        *timer += HZ_TO_US(60);
```

Comment:
Why introduce magic number as we have a define for it?

Reply:
I introduced TASK_LEDSTRIP_RATE_HZ but it was only used for rainbow effect. It is also used by @SteveCEvans optimization now which is much more appropriate than using it for rainbow. Check Larson implementation 60Hz is also used there.

Reply:
In that case consider creating another define - to not confuse with the one used by task rate

Reply:
Wait, I mixed it up with task rate, sorry.

Reply:
I think there is no need for a new define, task rate is only used now for changing task rate between normal and fast. It's less confusing now I think.

Reply:
Thanks for naming magic numbers.